src/body.js

```diff
@@ -0,0 +1,214 @@
+// this is multi-line code:
+
+function anotherExample(firstLine, lastLine) {
+  if (firstLine == '\`\`\`' && lastLine == '\`\`\`') {
+    return multiLineCode;
+  }
+}
+\`\`\`
+
+You can also make text **bold**... whoa!
+Or _italic_.
+Or... wait for it... **_both!_**
+And feel free to go crazy ~~crossing stuff out~~.
+
+There's also [links](https://www.freecodecamp.org), and
+> Block Quotes!
+
+And if you want to get really crazy, even tables:
+
+Wild Header | Crazy Header | Another Header?
+------------ | ------------- | -------------
+Your content can | be here, and it | can be here....
+And here. | Okay. | I think we get it.
+
+- And of course there are lists.
+  - Some are bulleted.
+     - With different indentation levels.
+        - That look like this.
+
+
+1. And there are numbered lists too.
+1. Use just 1s if you want!
+1. And last but not least, let's not forget embedded images:
+
+![freeCodeCamp Logo](https://cdn.freecodecamp.org/testable-projects-fcc/images/fcc_secondary.svg)
+`;
+
+const styleOne = {
+  display: 'none',
+};
+
+class Editor extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      textarea: placeholder,
+      editorElementmaximized: false,
+      previewElementMaximized: false,
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleEditorClick = this.handleEditorClick.bind(this);
+    this.handlePreviewClick = this.handleEditorClick.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      textarea: event.target.value,
+    });
+  }
+
+  handleEditorClick() {
+    this.setState({
+      editorElementmaximized: !this.state.editorElementmaximized,
+    });
+    console.log(this.state.editorElementmaximized);
+  }
+
+  handlePreviewClick() {
+    console.log('paola');
+    this.setState({
+      previewElementMaximized: !this.state.previewElementMaximized,
+    });
+  }
+
+  componentDidMount() {
+    document
+      .querySelector('.iconArrow1')
+      .addEventListener('click', this.handleEditorClick);
+    document
+      .querySelector('.iconArrow2')
+      .addEventListener('click', this.handlePreviewClick);
+  }
+
+  render() {
+    if (this.state.editorElementmaximized == true) {
+      document.querySelector('.container').classList.add('minContainer');
+      document.querySelector('.editor').classList.add('maxEditor');
+      document.querySelector('.containerPreview').style.display =
+        styleOne.display;
+    } else if (this.state.previewElementMaximized == true) {
+      document.querySelector('.editor').style.display = styleOne.display;
+    }
+
+    return (
+      <div className="container">
+        <div className="textTareaContainer">
+          <div className="barTextarea">
+            <FontAwesomeIcon
+              className="iconFreeCodeCamp"
+              icon={faFreeCodeCamp}
+            />
+            <p className="pBar">Editor</p>
+            <FontAwesomeIcon className="iconArrow1" icon={faMaximize} />
```

Comment:
in here we can use the `onClick` attribute, so it will be something like:
`<FontAwesomeIcon className="iconArrow1" icon={faMaximize} onClick={this.this.handleEditorClick} />`
and remove the code you have from line 89 - 94, because in **React** we try to don't interact with the dom directly